xml_inscriptions/transcribed/NY.NY.MMA.L.74.51.2203.xml

```diff
@@ -0,0 +1,179 @@
+
+                        <summary>
+                            <rs/>  <!-- content can be "modern" or blank -->
+                        </summary>
+
+                        <origin>
+                            <date evidence="#unspecified" notBefore="0070" notAfter="0120">ca.
+                                70-120 CE</date>
+                            <placeName ref="europe.cyprus">Cyprus</placeName>
+                        </origin>
+                        <provenance type="transferred" subtype="acquired">
+                            <p>The <persName role="collector"><name key="Cesnola">Cesnola</name></persName> Collection, Purchased by subscription, <date notBefore="1873" notAfter="1877">1874–76</date>.</p>
+                        </provenance>
+
+
+                    </history>
+                </msDesc>
+                <!--  *************************************  <listBibl>  *********************************  -->
+                <listBibl>
+                    <bibl>
+                        <ptr type="rest-of-bibl" target="#Myres"/>
+                        <biblScope unit="item">no. 2846</biblScope>
+                    </bibl>
+                </listBibl>
+            </sourceDesc>
+        </fileDesc>
+
+        <!--  *************************************  <encodingDesc>  ********************************* -->
+
+        <encodingDesc>
+
+                <xi:include
+                    href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+                    <xi:fallback>
+                        <p>Taxonomies for US Epigraphy controlled values</p>
+                    </xi:fallback>
+                </xi:include>
+
+        </encodingDesc>
+
+        <profileDesc> <!-- not used --> </profileDesc>
+
+
+        <!--  *************************************  <revisionDesc>  ********************************* -->
+    	<!-- Record all changes here -->
+
+        <revisionDesc>
+            <change when="2010-09-29" who="Center for Digital Scholarship">Converted to September 2009 EpiDoc</change>
+            <change when="2011-05-13" who="Elli Mylonas">More tweaks for bib only inscriptions</change>
+            <change when="2021-02-04" who="Itamar Levin">Transcription, translation, and
+                metadata</change>
+        </revisionDesc>
+
+    </teiHeader>
+
+    <!--  *************************************  <facsimile>  *********************************  -->
+        <!-- Used to record images of an inscription.   -->
+
+    <facsimile>
+        <surface>
+            <desc/>
+            <graphic url="NY.NY.MMA.L.74.51.2203-Det1.jpg"/>
+        </surface>    
+        <surface>
+            <desc/>
+            <graphic url="NY.NY.MMA.L.74.51.2203-Det2.jpg"/>
+        </surface>
+    </facsimile>
+    
+
+    <text>
+        <body>
+            <div type="edition">
+            <ab>
+                <lb n="1"/><persName><name key="Romansis">Romanesis</name></persName>.
```

Comment:
Key needs correction; also, include `<hi>` elements for the s characters that are reversed

Reply:
I've corrected myself.